### PR TITLE
Module 1- Hello Microverse Project

### PR DESCRIPTION
**Pull request by oluwatobi-fasasi**
- Ensured no linter errors.
- Added a .gitignore
- Added HTML file.
- Added CSS file.
- Created branches with different base branches.
- Followed GitHub flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,141 @@
...
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+- Getting Started
+- Authors
+- License
+<!-- TABLE OF CONTENTS -->
```
Comment (gemmen29) [inline README.md:4]: ### Kindly update your README with the following:
   - [x] All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. 
   - [ ] The only optional section is FAQ.
   - [ ] You should delete the live demo section if you decide not to deploy the application.
   - [ ] Make sure to add the section Future Features, including 1-3 potential features to be added.
   - [ ] You should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project.
   - [ ] It is your choice regarding how much into the details they go in each section of the README.
   - [ ] You should remove the **template logo**. Optionally, you can replace it with your own logo if you have one.
   - [ ] Kindly update the link to the MIT license at the bottom of README as now when I press it I get 404.
   - [ ] You don't need to include the objectives, general requirements, and requirements or a definition of the languages in the Built with section.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @oluwatobi-fasasi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @oluwatobi-fasasi, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (oluwatobi-fasasi) [reply README.md:4]: Thanks @gemmen29, all suggestions changes are fully implemented.

**Pull request merged**